modules/e2ee/OlmAdapter.js

```diff
@@ -427,7 +428,38 @@ export class OlmAdapter extends Listenable {
             break;
         }
         }
+    }
 
+    /**
+     * Handles a participant joining. Id e2ee is enabled,
+     * when a participant joines an olm session is initialized if participantId is
+     * "smaller" than the localParticipantId.
+     *
+     * @private
+     */
+    async _onParticipantJoined(id, participant) {
```

Comment:
Shouldn't we wait for them to turn on E2EE? It would be done by them, when they receive the e2ee.enabled property for any of the existing participants, but I don't see that code here.

Reply:
We do that on the app side. The joined participant will establish the session with every existing participant that has the id smaller and here we are just initialising the session with that participant if we have the smaller id

Reply:
That split doesn't sound right. Maybe we should just add a new config for toggling "auto E2EE".